packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -278,12 +278,14 @@ export class EdgelessPageBlockComponent
         </div>
         ${hoverRect} ${selectionRect}
         ${selectionState.type !== 'none'
-          ? html` <edgeless-selected-rect
-              .state=${selectionState}
-              .rect=${selectionState.rect}
-              .zoom=${zoom}
-              .readonly=${this.readonly}
-            ></edgeless-selected-rect>`
+          ? html`
+              <edgeless-selected-rect
+                .state=${selectionState}
+                .rect=${selectionState.rect}
+                .zoom=${zoom}
+                .readonly=${this.readonly}
```

Comment:
Not related to the PR, but should we access `readonly` by accessing `page.awareness.isReadonly`?

Reply:
We can have a new PR that removes the `readonly` prop.